### PR TITLE
build: upload plugin.zip after CI build

### LINKED ISSUE #921: Generate plugin archive on commit push
It'd be useful for non-developers to test latest changes from the main branch or a PR, directly without having to setup a full blown dev environment.
We can archive the plugins as part of a build, that power users can then install manually, by following [these instructions](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk).

### PR DESCRIPTION
**Pull request by fbricon**
Fixes #921

### CODE REVIEW

**Comment by fbricon** [comment]
See Quarkus Tools zip at the bottom of https://github.com/redhat-developer/intellij-quarkus/actions/runs/5191018965

File: README.md [hunk 1]
```diff
@@ -279,6 +279,14 @@ You can perform UI testing by running the following command:
 ```sh
 ./gradlew integrationTest 
 ```
+#### Testing the CI builds
+
+You can download and install CI builds of the latest commits or a pull request:
+
+- open the [`Java CI with Gradle` workflow](https://github.com/redhat-developer/intellij-quarkus/actions/workflows/ci.yml)
+- click on the build you are interested in
```
Comment (angelozerr) [inline README.md:287]: Please add a screenshot.

File: README.md [hunk 2]
```diff
@@ -279,6 +279,14 @@ You can perform UI testing by running the following command:
...
+- scroll down and download the `Quarkus Tools <version>.zip` file
```
Comment (angelozerr) [inline README.md:288]: Please add a screenshot.

File: README.md (hunk 1 above)
Comment (fbricon) [reply README.md:287]: let's keep the readme lean

File: README.md (hunk 2 above)
Comment (fbricon) [reply README.md:288]: let's keep the readme lean

**Comment by gastaldi** [comment]
I followed the steps in https://github.com/redhat-developer/intellij-quarkus#testing-the-ci-builds and tried installing the plugin produced in https://github.com/redhat-developer/intellij-quarkus/actions/runs/5199150324 but it fails with this error:

image

**Comment by fbricon** [comment]
@gastaldi my bad, you need to unzip the .zip.zip, than install the .zip

**Pull request merged**